python/ray/experimental/iter.py

```diff
@@ -238,6 +239,46 @@ def combine(self, fn: Callable[[T], List[U]]) -> "ParallelIterator[U]":
         it.name = self.name + ".combine()"
         return it
 
+    def local_shuffle(self, shuffle_buffer_size: int,
+                      seed: int = None) -> "ParallelIterator[T]":
+        """Remotely shuffle items of each shard independently
+
+        Arguments:
```

Comment:
```suggestion
        Args:
```

Could you make this same change everywhere in this file?